src/rules/property-blacklist/README.md

```diff
@@ -0,0 +1,55 @@
+# property-blacklist
+
+Specify a blacklist of disallowed properties.
+
+```css
+    a { 
+      color: pink;
+    }
+/**    ↑ 
+ * These properties */
+```
+
+## Options
+
+`array`: `"["array", "of", "unprefixed", "properties"]`
+
+### `["array", "of", "unprefixed", properties"]`
+
+Blacklisted properties *must never* be used.
+
+Given:
+
+```js
+["background-size", "transform"]
```

Comment:
>  It's a super simple rule, as you've shown, so why not have it in there and see if people make use of it.

Agreed :)

I think I'll switch out the examples to `animation` and `text-rendering`.